partner_firstname/models/base_config_settings.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingenieria S.L. - Antonio Espinosa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+from openerp import models, fields, api
+_logger = logging.getLogger(__name__)
+
+
+class BaseConfigSettings(models.TransientModel):
+    _inherit = 'base.config.settings'
+
+    partner_names_order = fields.Selection(
+        string="Partner names order",
+        selection="_partner_names_order_selection",
+        help="Order to compose partner fullname",
+        required=True,
+    )
+
+    def _partner_names_order_selection(self):
+        return [
+            ('last_first', 'Lastname Firstname'),
+            ('last_first_comma', 'Lastname, Firstname'),
+            ('first_last', 'Firstname Lastname'),
+        ]
+
+    def _partner_names_order_default(self):
+        return 'last_first'
```

Comment:
If you just add default='last_first' to the field definintion, you do not need this method nor calling it in get_default_partner_names_order.